vllm/transformers_utils/tokenizers/mistral.py

```diff
@@ -52,12 +52,13 @@ def __init__(self, tokenizer: PublicMistralTokenizer) -> None:
         assert isinstance(self.tokenizer,
                           (Tekkenizer, SentencePieceTokenizer)), type(
                               self.tokenizer)
-        self._is_tekken = isinstance(self.tokenizer, Tekkenizer)
 
-        if self._is_tekken:
+        if (is_tekken := isinstance(self.tokenizer, Tekkenizer)):
```

Comment:
Did you mean to include this change?

Reply:
Yes, this fixes a typing issue which failed mypy locally.